addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -360,6 +379,42 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 }
 
             }
+            // equalizer commands
+            if (channelId.equals(CHANNEL_EQUALIZER_BASS) || channelId.equals(CHANNEL_EQUALIZER_MIDRANGE)
```

Comment:
This `if` is about 100 lines -> perfect for its own method

Reply:
Done